src/main/java/com/google/sps/data/GivrUser.java

```diff
@@ -67,9 +73,15 @@ public boolean isLoggedIn() {
     return this.isLoggedIn;
   }
 
-  // TODO(): get correct moderator status for organization
-  public boolean isModerator(long organizationId) {
-      return false;
+  public ArrayList<Entity> getModeratingOrgs() {
+    return this.moderatingOrgs;
```

Comment:
I want you to think about how you can make sure getModeratingOrgs will always return the right thing. Right now the user may not have called setModeratingOrgs yet.

Reply:
My idea is to call setModeratingOrgs() before checking getModeratingOrgs().size(). Is there a better solution?

Reply:
Yup. Consider: do you want to do it here instead of in isModeratorOfAnyOrg?

Reply:
yes! That would be better, thank you! :D